flink-python/pyflink/fn_execution/beam/beam_coder_impl_slow.py

```diff
@@ -187,6 +187,24 @@ def __repr__(self):
         return 'ArrayCoderImpl[%s]' % repr(self._elem_coder)
 
 
+class PrimitiveArrayCoderImpl(StreamCoderImpl):
```

Comment:
I think we need to add related unit test of coders in test_coders.py and test_fast_coders.py